packages/web-shared/package.json

```diff
@@ -6,6 +6,8 @@
   "private": true,
   "dependencies": {
     "@babel/preset-react": "^7.22.5",
+    "@headlessui/react": "^1.7.18",
+    "polished": "^4.1.2",
```

Comment:
whoops - this leaked in. I also need it for the eventblock PR (which I'm opening on top of this). I can remove but it's just a bit of extra git steps. IF we're okauy with installing polished on this PR - that would save a few steps for me 😇

Reply:
I'm good with keeping it.